config/initializers/inflections.rb

```diff
@@ -0,0 +1,16 @@
+# Be sure to restart your server when you modify this file.
+
```

Comment:
One small comment (not a correction). When we start a new backend project we sometimes remove files that we know aren't gonna be used. For example, inflections.rb is only used if you want to tell Rails that the plural for a particular word is different than it expects (both for internationalization and for many of the things mentioned [here](https://gist.github.com/alexpchin/f5d2be2ef3735889d315) (ignore the test parts if you read this). So, unless the project includes a lot of uncommon words for model names and such, this file usually goes unused. Same goes for ActionCable classes unless we know we're gonna use ActionCable (don't worry about it now though). Keep it in mind when (if) starting new projects in the future.